Sources/SpyableMacro/Macro/SpyableMacro.swift

```diff
@@ -61,7 +61,11 @@ public enum SpyableMacro: PeerMacro {
 extension DeclSyntaxProtocol {
   /// - Returns: The preprocessor `flag` parameter that can be optionally provided via `@Spyable(flag:)`.
   fileprivate var preprocessorFlag: String? {
-    self.as(ProtocolDeclSyntax.self)?.attributes.first?
+    self
+      .as(ProtocolDeclSyntax.self)?.attributes.first {
```

Comment:
If a newline is not added after `self` the formatting becomes unpretty when running 'make format,' as shown in the attached screenshot, so I added the newline 🤔 
<img width="726" alt="스크린샷 2024-01-27 오전 4 13 51" src="https://github.com/Matejkob/swift-spyable/assets/50410213/fdd5e505-a331-448c-b239-7c1d366362cc">

Reply:
That's find, thank you! 🙌